### ISSUE TITLE
Serializer exception on movie not found

### ISSUE THREAD

**Issue by jellyfish77** [jellyfish77/camel-eip#2]
```
java.lang.NullPointerException
	at movies.OmdbMovieDeserializer.deserialize(OmdbMovieDeserializer.java:38)
	at movies.OmdbMovieDeserializer.deserialize(OmdbMovieDeserializer.java:18)
```

**Comment by jellyfish77** [comment]
Reproduce using 'Godzilla Resurgence'

**Comment by jellyfish77** [comment]
Fixed 5527829
